src/modules/gift/helpers/get-counterparty-urns/index.test.ts

```diff
@@ -0,0 +1,44 @@
+import { EXAMPLES } from '@ukef/constants';
+
+import { GiftFacilityCounterpartyDto } from '../../dto';
+import { getCounterpartyUrns } from '.';
+
+const {
+  GIFT: { COUNTERPARTY },
+} = EXAMPLES;
+
+const mockCounterparties: GiftFacilityCounterpartyDto[] = [COUNTERPARTY(), COUNTERPARTY(), COUNTERPARTY()];
```

Comment:
Is this a function `COUNTERPARTY()`?
Not convienced on the naming convention for a function.